app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsScreen.kt

```diff
@@ -126,17 +141,22 @@ private fun PullRequestsList(
     pullRequests: List<PullRequest>,
     modifier: Modifier,
     onItemClick: (PulLRequestsAction) -> Unit,
+    pullRefreshState: PullRefreshState,
+    isLoading: Boolean,
 ) {
-    LazyColumn(
-        modifier = modifier.fillMaxSize(),
-    ) {
-        itemsIndexed(pullRequests) { index, item ->
-            PullRequestItem(
-                index = index,
-                data = item,
-                onClick = onItemClick,
-            )
+    Box(modifier = Modifier.pullRefresh(pullRefreshState)) {
```

Comment:
**Praise:** I haven't seen the implementation of pull refresh in compose, it seems simple. So glad to see that 👌